controllers/yawol-controller/loadbalancer/loadbalancer_controller_test.go

```diff
@@ -248,6 +250,93 @@ var _ = Describe("loadbalancer controller", Serial, Ordered, func() {
 			})
 		})
 
+		It("should scale down old lbset after new one has ready keepalived", func() {
+			var oldLbs runtimeClient.ObjectKey
+			By("waiting for lbset creation")
+			hopefully(lbNN, func(g Gomega, act LB) error {
+				var lbsetList yawolv1beta1.LoadBalancerSetList
+				g.Expect(k8sClient.List(ctx, &lbsetList, &runtimeClient.ListOptions{
+					LabelSelector: labels.SelectorFromSet(lb.Spec.Selector.MatchLabels),
+				})).Should(Succeed())
```

Comment:
You can use `client.MatchingLabels` for brevity.

```suggestion
g.Expect(k8sClient.List(ctx, &lbsetList, runtimeClient.MatchingLabels(lb.Spec.Selector.MatchLabels))).Should(Succeed())
```